build/UIKit/lib/UIKitLib.vcxproj

```diff
@@ -110,6 +110,20 @@
     <ClangCompile Include="..\..\..\Frameworks\UIKit\UIView.mm" />
     <ClangCompile Include="..\..\..\Frameworks\UIKit\UIViewController.mm" />
     <ClangCompile Include="..\..\..\Frameworks\UIKit\UIWindow.mm" />
+    <ClangCompile Include="..\..\..\Frameworks\UIKit\WYPopoverController.m">
+      <ObjectiveCARC Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">true</ObjectiveCARC>
+      <ObjectiveCARC Condition="'$(Configuration)|$(Platform)'=='Debug|ARM'">true</ObjectiveCARC>
+      <ObjectiveCARC Condition="'$(Configuration)|$(Platform)'=='Release|ARM'">true</ObjectiveCARC>
+      <ObjectiveCARC Condition="'$(Configuration)|$(Platform)'=='Release|Win32'">true</ObjectiveCARC>
+      <OptimizationLevel Condition="'$(Configuration)|$(Platform)'=='Release|Win32'">Disabled</OptimizationLevel>
+      <OptimizationLevel Condition="'$(Configuration)|$(Platform)'=='Release|ARM'">Disabled</OptimizationLevel>
```

Comment:
Also in debug builds most/all of the problems in WYPopoverController were from calling a method returning a struct on a nil object. In fact, if I add back the unconditional messaging to WYPopoverController, the issue should show up in debug builds as well. Totally speculative but I wonder if clang is removing the nil checks under some bad assumptions about the ABI (more likely some other issue though).